src/sinks/amqp/config.rs

```diff
@@ -19,6 +19,9 @@ pub struct AmqpPropertiesConfig {
 
     /// Expiration for AMQP messages (in milliseconds)
     pub(crate) expiration_ms: Option<u64>,
+
+    /// Priority for AMQP messages.
+    pub(crate) priority: Option<u8>,
```

Comment:
Optimally, it would be possible to make this property a `Template` so it can be dynamically calculated per event. I could not find other examples where a `Template` is expected to output a `uint`. The `str` output of a `Template` could be parsed as a `uint`, but would be error-prone if the result is not numeric, or out-of-bounds.